microk8s-resources/wrappers/microk8s-stop.wrapper

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eu
+
+export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
+
+sudo snap stop ${SNAP_NAME} --disable
```

Comment:
As with `start`, can we point the user somewhere useful if `stop` returns non-zero?

Reply:
Done